front/lib/api/files/upsert.ts

```diff
@@ -293,6 +366,12 @@ const getProcessingFunction = ({
       } else {
         return undefined;
       }
+    case "application/vnd.dust.section-structured":
```

Comment:
Feels like we want a `json` in there?
Maybe `application/vnd.dust.section-json` ? or `application/vnd.dust.section.json` ? or `application/vnd.dust.section-structured.json` ?

cc @philipperolet does it feel aligned with the mimeTypes you introduced?

Reply:
Went with `application/vnd.dust.section.json`, can rename if needed!